### ISSUE TITLE
Default plotting example raises ValueError

### ISSUE THREAD

**Issue by Tim024** [poliastro/poliastro#1078]
I tried plotting the default example as follows:
```
from poliastro.plotting.static import StaticOrbitPlotter
from poliastro.bodies import Earth
from astropy import units as u
from poliastro.twobody import Orbit
import matplotlib.pyplot as plt
from astropy.time import Time


if __name__ == '__main__':
    r = [-6045, -3490, 2500] * u.km
    v = [-3.457, 6.618, 2.533] * u.km / u.s

    orbit = Orbit.from_vectors(Earth, r, v)
    op = StaticOrbitPlotter()
    op.plot(orbit,label=str(orbit.epoch))
    plt.show()
```

I get the following error:

```
Traceback (most recent call last):
  File "example.py", line 24, in <module>
    op.plot(orbital_gso.orbit,label=str(orbital_gso.orbit.epoch))
  File "/venv/lib/python3.8/site-packages/poliastro/plotting/static.py", line 215, in plot
    lines = self._plot(orbit, label=label, color=color, trail=trail)
  File "/venv/lib/python3.8/site-packages/poliastro/plotting/_base.py", line 153, in _plot
    label = generate_label(orbit.epoch, label)
  File "/venv/lib/python3.8/site-packages/poliastro/plotting/util.py", line 20, in generate_label
    epoch.out_subfmt = "date_hm"
  File "/venv/lib/python3.8/site-packages/astropy/time/core.py", line 622, in out_subfmt
    self._time.out_subfmt = val
  File "/venv/lib/python3.8/site-packages/astropy/time/formats.py", line 183, in out_subfmt
    self._select_subfmts(subfmt)
  File "//venv/lib/python3.8/site-packages/astropy/time/formats.py", line 396, in _select_subfmts
    raise ValueError(f'subformat not allowed for format {cls.name}')
ValueError: subformat not allowed for format jyear_str
```

I found that commenting out the line `epoch.out_subfmt = "date_hm"` in plotting/util.py line 20 makes everything work correctly but it is not convenient.

**Comment by Tim024** [comment]
Actually, this issue has been solved already in commit dd81e8a7fa1f44b38d7ffd1b6c2c83bd2d65ea51, I will close it.

**Comment by astrojuanlu** [comment]
That's it @Tim024! If you are using the stable poliastro version you can as well downgrade Astropy.
